### ISSUE TITLE
[Bug]: Android Auto automatic resume is intermittent

### ISSUE THREAD

**Issue by kanine** [advplyr/audiobookshelf-app#1309]
### What was the Problem?

The following setting in Android Auto should mean playback continues when media was playing when starting driving, picking up from the correct position in the book

### Steps to Reproduce the Issue

Download a book for offline playback (my preference).

1. Connection Preferences > Android Auto > Start music automatically > ON
2. Listen to a book whilst driving or parked
3. Leave vehicle, return later
4. Drive off and playback should resume from slightly behind where you left off

However this is happening probably less than 1/5 times. Generally I would have to then go into the app and then chose the book again from the downloaded section to resume.

### What was Expected?

The book should resume playback reliably and start off with a 5-10s rewind (give or take)

### Phone Model

Pixel 6 Pro

### Phone OS

Android 14

### Audiobookshelf App Version

Android App - 0.9.76

### Installation Source

Google Play Store

### Additional Notes

Tried rebooting and working with a server that is online, but even the online server seems quite hit or miss on resume too.

Due to this issue I have been using SAP, which performs this function perfectly every time.

**Comment by ISO-B** [comment]
I did some testing to this and noticed that if Audibookshelf app is open on background then Android Auto seems to continue playing book reliable. When app is not open Android Auto doesn't continue playing book and wont show what book I was listening last. 

One thing I noticed that offline books wont show covers on Android auto widget or player, but cover is visible on listings. Have you encountered this behavior?

**Comment by freitagdavid** [comment]
I recently updated to a pixel 9 and it seems audiobookshelf isn't seen as a media player at all from android auto at this point. If I'm playing an audiobook it doesn't show playback controls, it can't adjust the volume, and it doesn't resume it always picks synfonium or yt music. Whichever was most recently used.

I have been able to workaround these issues but they are definitely on the annoying side. I'm tempted to take a look at them myself, where would I find the stuff related to android auto? Or is it just something that naturally comes with the app being seen as a media player? If so where would I find where it's defined as a media player?

**Comment by gazpachoking** [comment]
Yeah, I have this problem a lot. When I get in to the car, the book very rarely starts playing again, and is not very often in my 'continue playing' list either. Other ABS books show up there, just not the one I'm actually listening to.

> I recently updated to a pixel 9 and it seems audiobookshelf isn't seen as a media player at all from android auto at this point.

I have a Pixel 9 pro and have never seen this.
